client/src/components/TimeChart.vue

```diff
@@ -62,17 +62,26 @@ export default {
           backgroundColor: '#009ec367',
           label: that.datumname,
           data: that.chartData,
-          cubicInterpolationMode: 'monotone'
+          cubicInterpolationMode: 'monotone',
+          spanGaps: true
         }]
       },
       options: {
+        responsive: true,
         annotation:{
           annotations:[{
+            label:
+              {
+                enabled: true,
+                position: 'left',
+                content: (this.limit_direction === 'min') ? 'minimum' : '',
```

Comment:
This label is super distracting, I think it would be a lot better as a legend.

Reply:
I don't quite think this is supported by the plugin that we use. ChartJS obviously has legends,(https://www.chartjs.org/docs/latest/configuration/legend.html) but I don't see any support for legends in the ChartJS annotation plugin. What I could maybe try doing is only showing the annotation label if you hover over the chart?